global.json

```diff
@@ -1,6 +1,6 @@
 {
     "sdk": {
-        "version": "8.0.100",
+        "version": "8.0.404",
```

Comment:
This change is probably good @martinothamar. Or is there a reason you didn't require everyone to upgrade to latest?

Reply:
just wanted to be inclusive for people hanging on to older SDKs ;) personally I'm fine with this